### ISSUE TITLE
Adding github issues broken

### ISSUE THREAD

**Issue by prestonp** [goodybag/trello-to-github#14]
![screen shot 2013-11-07 at 11 01 52 am](https://f.cloud.github.com/assets/1265379/1493531/64484040-47ce-11e3-93e8-039068a5543d.png)

**Comment by ZaLiTHkA** [comment]
I'm having similar trouble, but I don't get an alert message like that. My console logs do start off with `gapi is not defined` though.

Before I open a new issue (I see the post date), was this particular one ever solved..?

**Comment by jrf0110** [comment]
We never did resolve this issue. I'm sure it's a super easy fix if you want to give fixing it a shot. I'm sure @ProCynic will publish a new version if you figure it out :D

**Comment by ZaLiTHkA** [comment]
Nope, sorry.. Keep in mind I have no experience working on Chrome extensions.

However, even after removing the extension, I'm still getting the same error. I suspect that's _causing_ an issue for this extension though, rather than being caused by it.
